data/hai/hai reveal 1 intro.txt

```diff
@@ -524,12 +532,26 @@ mission "Hai Leaks Response 1"
 			`	You haven't really seen very much of the Hai so far, and you can't help but think the size of the detachment is rather militant of them. It's impossible to judge if it's for your safety or whether, given your status as a "war hero", it's for theirs.`
 			`	Word of your status in human space also seems to have spread ahead of you, perhaps because you are a significant human visitor in the current climate. A small crowd of Hai gather to peek at you on the short walk from the spaceport to the government building you are led to.`
 			label resume
+			choice
+				`	(Keep silent and observe.)`
+					goto observe
+				`	(Ask about the large security detachment.)`
+			`	You turn to the Hai with the most decorated uniform and ask, "Why do I have so many escorts?"`
+			`	"Crowd control," the Hai responds. "You've become the center of attention of politicians in a media circus. Life in Hai space is complicated of late, and we want to make sure you are not stopped by crowds with endless questions before reaching your meeting."`
```

Comment:
```suggestion
			`	He looks back to the crowd for a moment. "Life in Hai space is complicated of late, and we want to make sure you are not stopped by crowds with endless questions before reaching your meeting. It would be... inefficient."`
```